packages/db/mocks/vote.mock.ts

```diff
@@ -1,8 +1,10 @@
-import { type Prisma } from '@prisma/client';
 import { faker } from '@faker-js/faker';
+import { type Prisma } from '@prisma/client';
 
 export function VoteMock(userIds: string[]): Prisma.VoteCreateManyInput {
   return {
     userId: faker.helpers.arrayElement(userIds),
+    rootType: 'CHALLENGE',
+    challengeId: 2,
```

Comment:
Same thing as the comment.mock.ts i made -- you can definitely rig up the seed script to pull a random challenge ID so that challenge 2 is the only one with things.

Reply:
hmmm ok, I'll do this in a separate PR

Reply:
i hard coded 2 so i could always just point to one challenge to test things 😀